run_node.sh

```diff
@@ -130,3 +139,72 @@ then
                -d ${ALEPH_IMAGE}
 fi
 
+echo 'Performing session key checks...'
+
+if [[ -n "$ARCHIVIST" ]]
+then
+    echo 'Node run as archivist: no need to check session keys.'
```

Comment:
We might (and this is mostly thinking out loud) want to replace the 'archivist' with 'rpc node', the name archivist is a little misleading